src/main/java/org/zalando/nakadi/repository/db/EventTypeCache.java

```diff
@@ -146,23 +172,19 @@ public EventTypeValidator getValidator(final String name) throws InternalNakadiE
                 .orElseThrow(() -> new NoSuchEventTypeException("Event type " + name + " does not exists"));
     }
 
-    private static PathChildrenCache setupCacheSync(final CuratorFramework zkClient) throws Exception {
-        try {
-            zkClient.create()
-                    .creatingParentsIfNeeded()
-                    .withMode(CreateMode.PERSISTENT)
-                    .forPath(ZKNODE_PATH);
-        } catch (final KeeperException.NodeExistsException expected) {
-            // silently do nothing since it means that the node is already there
-        }
-
-        final PathChildrenCache cacheSync = new PathChildrenCache(zkClient, ZKNODE_PATH, false);
-
-        // It is important to preload all data before specifying callback for updates, because otherwise preload won't
-        // give any effect - all changes will be removed.
-        cacheSync.start(PathChildrenCache.StartMode.BUILD_INITIAL_CACHE);
+    public Optional<Timeline> getActiveTimeline(final String name) throws InternalNakadiException,
+            NoSuchEventTypeException {
+        return getCached(name).map(CachedValue::getTimelines)
```

Comment:
Wouldn't it be better to store the active timeline directly so not to map+filter every time this method is called?